package.json

```diff
@@ -29,20 +31,18 @@
   "homepage": "https://github.com/mostjs/prelude#readme",
   "devDependencies": {
     "@most/eslint-config-most": "^1.0.2",
-    "babel-cli": "^6.5.0",
-    "babel-core": "^6.5.0",
-    "babel-plugin-transform-es2015-modules-umd": "^6.5.0",
-    "babel-preset-es2015": "^6.5.0",
     "babel-preset-power-assert": "^1.0.0",
+    "buba": "^2.0.1",
     "eslint": "^2.3.0",
     "eslint-config-standard": "^5.1.0",
     "eslint-plugin-promise": "^1.1.0",
     "eslint-plugin-standard": "^1.3.2",
-    "isparta": "^4.0.0",
+    "istanbul": "^1.1.0-alpha.1",
     "jsinspect": "^0.7.2",
     "mocha": "^2.4.5",
     "power-assert": "^1.4.1",
-    "rollup": "^0.25.3",
+    "rollup": "^0.25.8",
```

Comment:
Can we use the newest rollup, or do we need to stick with an older version?  (0.34.7 as of right now)